core/domain/story_services.py

```diff
@@ -268,8 +268,14 @@ def apply_change_list(story_id, change_list):
                 story.delete_node(change.node_id)
             elif (change.cmd ==
                   story_domain.CMD_UPDATE_STORY_NODE_OUTLINE_STATUS):
-                story.update_node_outline_status(
-                    change.node_id, change.finalized)
+                if change.new_value:
+                    story.mark_node_outline_as_finalized(change.node_id)
+                elif not change.new_value:
+                    story.mark_node_outline_as_unfinalized(change.node_id)
+                else:
+                    raise Exception(
```

Comment:
Whether boolean or not, the above two cases will cover all possibilities. So you might as well just do "if" and "else".

Reply:
Done.